sharedbuffers/mapped_struct.py

```diff
@@ -1027,24 +1038,28 @@ def unpack_from(cls, buf, offs, idmap = None, packer = struct.Struct('=q')):
         return rv
 
 class mapped_datetime(datetime):
+    PACKER = struct.Struct('=q')
+
     @classmethod
     @cython.locals(offs = cython.longlong, implicit_offs = cython.longlong, timestamp = cython.longlong)
-    def pack_into(cls, obj, buf, offs, idmap = None, implicit_offs = 0, packer = struct.Struct('=q')):
+    def pack_into(cls, obj, buf, offs, idmap = None, implicit_offs = 0):
```

Comment:
Why is this change only done for the datetime and not for other classes (like the date one)?

Reply:
Um...